src/services/millions-draw.service.ts

```diff
@@ -13,14 +13,23 @@ export class MillionsDrawService {
         return this._repository;
     }
 
-    getById = async (id: string): Promise<MillionsDrawEntity> => {
+    getById = async (id: string): Promise<MillionsDrawEntity | null> => {
         return this._repository.findOne({ where: { id } });
     };
 
     exist = async (id: string): Promise<boolean> => {
         return this._repository.exist({ where: { id } });
     };
 
+    existMoreThan = async (id: string, timeInSec: number): Promise<boolean> => {
+        const date = new Date();
```

Comment:
Shouldn't we use `dayjs` package here, in order to consistent with the way we handle dates across the repo ?
e.g `asset.scheduler`, `block.consumer`, `millions.scheduler`